.github/workflows/golangci-lint.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@1481404843c368bc19ca9406f87d6e0fc97bdcfd # v7.0.0
         with:
-          version: v2.0
+          version: v2.1.2
```

Comment:
Can be `v2.1` which should pull the latest patch release for a given minor version

Reply:
changed.